cpp/ql/test/query-tests/Likely Bugs/Underspecified Functions/test.c

```diff
@@ -30,8 +30,8 @@ void test(int *argv[]) {
   
   not_yet_declared1(1); // BAD (GOOD for everything except for cpp/implicit-function-declaration)
   not_yet_declared2(1); // BAD (GOOD for everything except for cpp/implicit-function-declaration)
-  not_yet_declared2(ca); // BAD
-  not_yet_declared2(); // BAD
+  not_yet_declared2(ca); // BAD (GOOD for everything except for cpp/mistyped-function-arguments)
+  not_yet_declared2(); // GOOD
```

Comment:
I think changing this to "GOOD" is incorrect. It's still "BAD", but we no longer detect it. For that we have "BAD [NOT DETECTED]". That probably does need some qualification here, as that only applies to cpp/too-few-arguments.